cosmos_rl/rollout/worker/colocated/rollout_control.py

```diff
@@ -66,6 +66,8 @@ def policy_to_rollout_unicast(self, command: PolicyToRolloutUnicastCommand):
         if command.dst_replica_name != self.replica_name:
             return
         self.rollout.set_underlying_model(self.api_client.get_policy_model())
+        if self.get_underlying_model() is not None:
+            self.get_underlying_model()._set_fsdp_reshard_after_forward("never")
```

Comment:
We may add a such setting to make the reshard strategy back to `self.config.train.reshard_after_forward` after each rollout generation or before collocated trainer control?  Although we have a setting inside vla trainer case, but for non-VLA case, this would break?